README.md

```diff
@@ -72,4 +72,102 @@ to GHCR. In order to accomplish this task:
+    start_date: "2024-04-09T00:00:00"
+    catchup: false
+    default_args:
+      retries: 2
+    tags:
+      - "nextflow_tower"
+```
+
+----
+
+### Overview
+
+As part of propping up a new Synapse-hosted challenge, the workflow you choose in [nf-synapse-challenge](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/tree/main) will need to
+be automated to run for challenge submissions on a rolling basis. For this we leverage Airflow DAGs to perform these
+automated executions of your customized challenge workflow.
+
+For introducing a challenge DAG, there are two files that come into play:
+
+1. The `challenge_configs.yaml` file, which holds configuration settings for each challenge DAG. This file allows you to easily add
+   new challenges or adjust existing ones without modifying any Python DAG scripts directly.
+   The configurations in this file are used by the DAG factory to dynamically create DAGs.
+
+2. The `challenge_dag_factory.py` is a Python script that:
+   * Loads challenge configurations from the YAML file.
+   * Resolves the `dag_config`, that is, the configuration settings needed to generate the DAG dynamically at runtime.
+   * Generates runtime parameters (such as a unique run-specific UUID) and injects them into the DAG.
+   * Creates and wires tasks together to fetch submissions, update statuses, generate CSV manifests, launch workflows, and monitor execution.
+
+By updating `challenge_configs.yaml` with your **challenge DAG config**, you can contribute new challenges without needing to change the underlying DAG generation code.
+
+----
+
+### Creating Your Challenge DAG Config
+
+Each challenge DAG config will have its own set of parameters depending on the customized workflow profile on [nf-synapse-challenge](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/blob/main/nextflow.config) and what part of Synapse and AWS S3 the workflow will
+be communicating with.
+
+See below for a list of parameters and their descriptions:
+
+#### Parameters
+
+1. `synapse_conn_id`: The Airflow connection ID used to connect to the Synapse service. This connection is utilized when fetching submission data and updating statuses. **Use `SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN`**.  
+1. `aws_conn_id`: The connection ID for AWS. This enables the DAG to upload CSV files (manifests) to an S3 bucket via the S3 hook. **Use `AWS_TOWER_PROD_S3_CONN`**.  
+1. `revision`: Specifies the version or Git commit revision of the `main` branch in the `nf-synapse-challenge` repository (which houses your workflow). This ensures that the correct version of your workflow is deployed when the DAG triggers a run. **Use `main` or a specific commit SHA which points to the desired version**.  
+1. `challenge_profile`: Identifies the Nextflow Tower challenge profile you contributed in `nextflow.config` of the `nf-synapse-challenge` repository. This parameter customizes the execution environment for the workflow.  
```

Comment:
Should there be an example of a profile or a link to an example PR where a profile was added to the Nextflow workflow?